CameraXBasic/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -57,6 +57,9 @@
         <argument
             android:name="root_directory"
             app:argType="string" />
+        <action
+            android:id="@+id/action_gallery_fragment_to_camera_fragment2"
+            app:destination="@id/camera_fragment" />
```

Comment:
This is unnecessary if you perform up navigation as suggested in the other comment.

Reply:
Sir crash resolved from this act

Reply:
See above comment, this change is unnecessary